### ISSUE TITLE
error thrown when files do not require ava

### ISSUE THREAD

**Issue by GabiGrin** [sindresorhus/ava#209]
When running AVA against empty files or files that do not require ava, this error is thrown on v.0.4.2.
I tried with master, and it just hangs up.
getting the same error as in #198 (I opened a new issue because I'm not sure it's the same cause.):
``` 
TypeError: Cannot read property 'stats' of undefined
    at C:\Users\Frontend\AppData\Roaming\nvm\v0.12.5\node_modules\ava\cli.js:139:16
    at Array.map (native)
    at exit (C:\Users\Frontend\AppData\Roaming\nvm\v0.12.5\node_modules\ava\cli.js:138:22)
From previous event:
    at Object.<anonymous> (C:\Users\Frontend\AppData\Roaming\nvm\v0.12.5\node_modules\ava\cli.js:216:18)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3 
```

In master, I saw that if the file transpiled by babel does is not processed by AVA and thus emits the needed events to end, the process will just hang up.

**Comment by sindresorhus** [comment]
> When running AVA against empty files or files that do not require ava

We're going to fix it, but I have to ask, why are you doing that?

**Comment by GabiGrin** [comment]
It wasn't intentional :)
On my first attempt to use AVA I ran it on a folder with another empty file and got into this issue. I thought that debugging and trying to fix it will be a nice way to understand how AVA works.

**Comment by jamestalmage** [comment]
Fix landed in @97409864e, you should be good to go on next release
